### ISSUE TITLE
enemies

### ISSUE THREAD

**Issue by RevenReborn** [christopho/solarus-alttp-pack#8]
hey, would it be possible to upload some enemies from thew Solarus team projects like the Mystery of Solarus games? I'm desperate for some new enemies and can't find any to download anywhere, Zelda or otherwise.

**Comment by christopho** [comment]
Yes it will be done at some point. Sorry for the delay. In the meantime, you can copy the enemies from Zelda Mystery of Solarus: https://github.com/solarus-games/zsdx and also from Zelda Return of the Hylian SE: https://github.com/solarus-games/zelda-roth-se

**Comment by RevenReborn** [comment]
Oh thanks. I'll do that then, I've gotten tired with green blobs as the
only enemy in my games. Thanks.

**Comment by mannywald** [comment]
They work well!

If you run into issues then you may need to comment out the call to enemy:set_random_treasures in some enemy scripts (or implement the function yourself/import it).
